server/rpc/rpcwebsocket.go

```diff
@@ -361,6 +361,12 @@ func (f *wsClientFilter) existsUnspentOutpoint(op *wire.Outpoint) bool {
 	return ok
 }
 
+func (f *wsClientFilter) existsUnspentOutpointWithLock(op *wire.Outpoint) bool {
```

Comment:
We agreed some time ago that the standard for functions with that have lock/no-lock version is xxx and xxxNoLock.
Lets keep it this way.